### PR TITLE
Fix onMessageReactionEvent

### PR DESCRIPTION
**Pull request by bsian03**

### CODE REVIEW

File: lib/command/CommandClient.js [hunk 1]
```diff
@@ -210,7 +210,8 @@ class CommandClient extends Client {
         }
     }
 
-    async onMessageReactionEvent(msg, emoji, userID) {
+    async onMessageReactionEvent(msg, emoji, reactor) {
+        const userID = reactor.id || reactor;
```
Comment (alex-taxiera) [inline lib/command/CommandClient.js]: the types indicate that `reactor` will be either `Member` or `{ id: string }`. is the `|| reactor` necessary?

Comment (bsian03) [reply lib/command/CommandClient.js]: Yes, reaction remove only passes the user ID

Comment (alex-taxiera) [reply lib/command/CommandClient.js]: Cool, then the types should be adjusted to reflect that :^)

Comment (alex-taxiera) [reply lib/command/CommandClient.js]: `reactor` in `messageReactionAdd` is `Member | { id: string })`

but here `reactor` in `onMessageReactionEvent` is `Member | { id: string }) | string`

Comment (bsian03) [reply lib/command/CommandClient.js]: Yeah and `messageReactionRemove` is `string`, so what's currently there is correct

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**